src/Illuminate/Foundation/Console/ViewCreateCommand.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Illuminate\Console\GeneratorCommand;
+use Illuminate\Support\Str;
+
+class ViewCreateCommand extends GeneratorCommand
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'make:view {name}
+                {--f|force : Overwrite existing view if any}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Create a view file in the resources/views directory';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+
+        // Here, we look at whether there is a view file to be created, then the relevant
+        // answers are returned to those who use the command, and at the end, the view file is
+        // created.
+
+        $force = $this->option('force');
+
+        // If it specifies directory and file as dots, it converts dots to slash(/)
+        $viewName = $this->getView();
```

Comment:
Personally i don't think you should extract this code to a separate method, since it's just a one line string manipulation.